tests/share/harvesters/test_swbiodiversity_harvester.py

```diff
@@ -0,0 +1,148 @@
+from datetime import timedelta
+
+import requests_mock
```

Comment:
Could you switch this to use httpretty? @laurenbarker and I thought the interface for it was a bit nicer. It would be good to standardize on a single library.